pyresample/geometry.py

```diff
@@ -1024,8 +1024,9 @@ class AreaDefinition(BaseDefinition):
         Human-readable description of the area
     proj_id : str
         ID of projection
-    projection: dict or str
-        Dictionary or string of Proj.4 parameters
+    projection: dict or str or pyproj.crs.CRS
+        Dictionary of PROJ parameters or string of PROJ or WKT parameters.
+        Can also be a :class:`pyproj.crs.CRS` object.
```

Comment:
Should the supported WKT version(s) be mentioned?

Reply:
I don't *think* so. I think it accepts any version of WKT and passes it to PROJ to decipher. Let me test it.

Reply:
Ok I did a few tests where I basically did:

```
CRS(crs.to_wkt(version='WKT1_GDAL'))
```

And other versions and it seems to work fine. Of course I suppose there could be a backwards incompatibility in the WKT format where someone passes a WKT1 string and it gets parsed as WKT2 and fails, but that would be pretty bad design on the PROJ/WKT folks part. I'm declaring this a non-issue, but feel free to push back if you are concerned.